rocketpy/environment/environment.py

```diff
@@ -2737,6 +2743,96 @@ def decimal_degrees_to_arc_seconds(angle):
         arc_seconds = (remainder * 60 - arc_minutes) * 60
         return degrees, arc_minutes, arc_seconds
 
+    def to_dict(self, include_outputs=False):
+        env_dict = {
+            "gravity": self.gravity,
+            "date": self.date,
+            "latitude": self.latitude,
+            "longitude": self.longitude,
+            "elevation": self.elevation,
+            "datum": self.datum,
+            "timezone": self.timezone,
+            "_max_expected_height": self.max_expected_height,
+            "atmospheric_model_type": self.atmospheric_model_type,
+            "pressure": self.pressure,
+            "barometric_height": self.barometric_height,
+            "temperature": self.temperature,
+            "wind_velocity_x": self.wind_velocity_x,
+            "wind_velocity_y": self.wind_velocity_y,
+            "wind_heading": self.wind_heading,
+            "wind_direction": self.wind_direction,
+            "wind_speed": self.wind_speed,
```

Comment:
These are also outputs, no?
```suggestion
```

Do not accept the suggestion directly, remember to add the attributes in the includ_outputs section

Reply:
They are necessary to rebuild the `environment` without going back to the API IIRC, aren't they?

Reply:
I don't know about that, if you have the wind_y and wind_x, shouldn't we be able to calculate the wind direction, speed and heading? I guess that depends on the way you are rebuilding the Environment object.

All in all, it is late here, so I don't mind keeping the way it is. Let's go.